src/main/java/javax/resource/spi/Activation.java

```diff
@@ -23,7 +23,7 @@
 import static java.lang.annotation.RetentionPolicy.*;
 
 /**
- * Designates a JavaBean as an <code>ActivationSpec</code>. This annotation may
+ * Designates a Java&trade; Bean as an <code>ActivationSpec</code>. This annotation may
```

Comment:
"JavaBean" is a single word.  Leave out the trademark.

Reply:
Done